# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/storage/SessionStorageManager.h
# Commit: fd389138a684
# Full Hash: fd389138a684c35685d93bf9c7e976aba106cdab
# Author: Yaron Tausky <ytausky@mozilla.com>
# Date: 2019-12-09 21:50:19
# Regressor Bug: 1593246
# File Overlap Count: 1
# Description:
#   Bug 1593246 - Part 3: Remove unused parameter from SessionStorageManager::GetSessionStorageCacheHelper r=dom-workers-and-storage-reviewers,sg
#   
#   Depends on D55659
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55660
# ==============================================================================

diff -r 2663311a1b62 -r fd389138a684 dom/storage/SessionStorageManager.h
--- a/dom/storage/SessionStorageManager.h	Thu Dec 05 17:51:59 2019 +0000
+++ b/dom/storage/SessionStorageManager.h	Thu Dec 05 17:52:44 2019 +0000
@@ -51,7 +51,6 @@
                      const nsACString& aOriginScope);
 
   nsresult GetSessionStorageCacheHelper(nsIPrincipal* aPrincipal,
-                                        nsIPrincipal* aStoragePrincipal,
                                         bool aMakeIfNeeded,
                                         SessionStorageCache* aCloneFrom,
                                         RefPtr<SessionStorageCache>* aRetVal);
